scvi/models/log_likelihood.py

```diff
@@ -7,12 +7,32 @@
 from torch.distributions import Normal
 
 
-def compute_log_likelihood(vae, posterior, **kwargs):
-    """ Computes log p(x/z), which is the reconstruction error .
-        Differs from the marginal log likelihood, but still gives good
-        insights on the modeling of the data, and is fast to compute
+def compute_elbo(vae, posterior, **kwargs):
+    """ Computes the ELBO.
+
+    The ELBO  is the reconstruction error + the KL divergences
+    between the variational distributions and the priors.
+    Differs from the marginal log likelihood, but still gives good
```

Comment:
You might say that the ELBO is a lower bound on the marginal log likelihood plus a term that is constant with respect to the variational distribution.

Reply:
Thanks, i'll put that in.